README.adoc

```diff
@@ -48,33 +47,31 @@ compiler currently runs on the following platforms:
 
 Tier 1 (actively used and maintained by the core OCaml team):
 
-AMD64 (Opteron)::    Linux, OS X, MS Windows
-IA32 (Pentium)::     Linux, FreeBSD, MS Windows
-PowerPC::            Linux, OS X
-ARM::                Linux
+x86 64 bits (AMD64)::    Linux, FreeBSD, MS Windows
+x86 32 bits (IA32)::     Linux, MS Windows
+POWER and PowerPC::      Linux
+ARM 32 bits::            Linux
+ARM 64 bits (AArch64)::  Linux
 
 Tier 2 (maintained when possible, with help from users):
 
-AMD64::              FreeBSD, OpenBSD, NetBSD
```

Comment:
If we remove OSX from tier 1, it should at least be here in tier 2/ADM64.

Reply:
That was a typo.  macOS is Tier 1, of course.